index.html

```diff
@@ -41,7 +41,180 @@
             </div>
         </div>
         </section>
-        
     </div>
+    <section class="works">
+        <p id="Recent-Works-Text">My Recent<br>Works</p>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
```

Comment:
I suggest you use a `<hx>` element, from` <h2>` to `<h6>` since there can only be one `<h1> `element in a page for your page title. This is for semantic purposes and so that screen readers can understand your page.
This is a heading of the individual cards